src/otx/cli/utils/__init__.py

```diff
@@ -2,3 +2,24 @@
 # SPDX-License-Identifier: Apache-2.0
 
 """CLI Utils."""
+
+import importlib
+import inspect
+from pathlib import Path
+
+
+def get_otx_root_path() -> str:
+    """Return the root path of the otx module.
+
+    Returns:
+        str: The root path of the otx module.
+
+    Raises:
+        ModuleNotFoundError: If the otx module is not found.
+    """
+    otx_module = importlib.import_module("otx")
+    if otx_module:
+        file_path = inspect.getfile(otx_module)
+        return str(Path(file_path).parent)
+    msg = "Cannot found otx."
```

Comment:
how about something liek cannot find otx in {path}, so when we see this error, we would also see what location we would look

Reply:
This is a function that finds the path based on the otx module in the environment where otx is installed, so I'm not sure which path it should show.